docs/next/pages/getting-started/install.md

```diff
@@ -22,11 +20,11 @@ pip --version
 
 ## Installing Dagster into an existing Python environment
 
-<Note>
-  <strong>Note</strong>: We strongly recommend installing Dagster inside a
-  Python virtualenv. If running Anaconda, install Dagster inside a Conda
-  environment.
-</Note>
+{% note %}
```

Comment:
❤️

Reply:
I'm having some trouble testing this locally (probably an env issue with the markdoc stuff), but d'you know if the new notes/callouts will preserve line breaks between paragraphs? Ex:

```
{% note %}
One line!

Another line!

Woo!
{% /note %}
```

This looks fine when rendered even though the migrated copy is on three lines (why is that, btw?), but just curious about multiple paragraphs being preserved.

Reply:
```
{% note %}

One line!

Another line!

Woo!

{% /note %}
```
would render as this:
<img width="631" alt="image" src="https://github.com/dagster-io/dagster/assets/4531914/44df0af2-2599-4338-af75-7bdf042d236a">

---

the migrated copy is on three lines but without empty new lines between each line, so it renders as one line. it's exactly how it was before which i guess was just a legacy formatting result.

Reply:
excellent, ty!